assets/js/googlesitekit/widgets/components/URLSearchWidget.js

```diff
@@ -30,9 +30,10 @@ import { CORE_SITE } from '../../datastore/site/constants';
 import { CORE_LOCATION } from '../../datastore/location/constants';
 import Button from '../../../components/Button';
 import PostSearcherAutoSuggest from '../../../components/PostSearcherAutoSuggest';
+import WidgetNull from './WidgetNull';
 const { useSelect, useDispatch } = Data;
 
-function URLSearchWidget( { Widget } ) {
+function URLSearchWidget( { Widget = WidgetNull } = {} ) {
```

Comment:
Looks you've got a working typo here. Also, no need to default to empty object since props will always be passed.
```suggestion
function URLSearchWidget( { Widget, WidgetNull } ) {
```

Reply:
It wasn't so much a typo, the props weren't being passed from /assets/js/googlesitekit/widgets/register-defaults.js, but I think that was more a symptom of the issue we were debugging. No need to destructure WidgetNull since we don't render it.